.travis.yml

```diff
@@ -9,6 +9,7 @@ jdk:
   - openjdk8
 
 script:
+  - mvn javadoc:javadoc
```

Comment:
I don't think it's ideal to build documentation on every commit, can you restrict it to master or remove it entirely ?

Reply:
I think the new step will prevent merging PRs with a failing javadoc without adding overhead. The step doesn't take more than a minute, but if we remove it, the javadoc check is done when uploading to maven. Therefore, I removed the new step. We can add it later if the issue appears again.